mocked-crates/futures-timer/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "futures-timer"
+version = "3.0.2"
+authors = ["Mysten Labs"]
+edition = "2018"
```

Comment:
version & edition?

Reply:
fixed edition - version must match the version that we are patching.